docs/spec/WIP_provisioning/overview.md

```diff
@@ -0,0 +1,229 @@
+# Fee Distribution
+
+## Overview
+
+Fees are pooled separately and withdrawn lazily, at any time.  They are not
+bonded, and can be paid in multiple tokens.  An adjustment factor is maintained
+for each validator and delegator to determine the true proportion of fees in
+the pool they are entitled too.  Adjustment factors are updated every time a
+validator or delegator's voting power changes.  Validators and delegators must
+withdraw all fees they are entitled too before they can bond or unbond Atoms.
+
+## Affect on Staking
+
+Because fees are optimized to note
+
+Commission on Atom Provisions and having atoms autobonded are mutually
+exclusive (we can’t have both). The reason for this is that if there are atoms
+commissions and autobonding, the portion of atoms the fee distribution
+calculation would become very large as the atom portion for each delegator
+would change each block making a withdrawal of fees for a delegator require a
+calculation for every single block since the last withdrawal. Conclusion we can
```

Comment:
Get rid of the line starting with "Conclusion we can...", not necessary.

Reply:
WIP, next PR